### PR TITLE
Update app.properties with required details

### LINKED ISSUE #21: [CHORE] Update app.properties with required details
**Describe the bug**
app.properties has not been updated.

**To Reproduce**
Update app.properties with required details using the template on this link(https://github.com/BuildForSDG/.github/blob/master/app.properties)

### PR DESCRIPTION
**Pull request by Yasholma**
## Description
Update app.properties with required details

Delivers #19 

## How Has This Been Tested?
check app.properties on root directory

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app.properties [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
-project.team.slackname:
-
-project.fb.opensource:
-
-program.cohort:1
\ No newline at end of file
+# about the product
+# ================= 
+# name of the product 
+name: SmeVest
```
Comment (oyewoas) [inline app.properties:4]: @Yasholma do you know why we have this ```^M```?

Comment (Yasholma) [reply app.properties:4]: no

Comment (Yasholma) [reply app.properties:4]: Is it version?

**Review by oyewoas — APPROVED** [review]
Nice @Yasholma

**Pull request merged**